src/util/mount.c

```diff
@@ -79,19 +121,29 @@ int check_mounted(char *mountpoint) {
         mountpoint[mountpoint_len-1] = '\0';
     }
 
-    real_mountpoint = realpath(mountpoint, NULL); // Flawfinder: ignore
+    real_mountpoint = realpath(joinpath(rootfs_dir, mountpoint), NULL); // Flawfinder: ignore
     if ( real_mountpoint == NULL ) {
-        // mountpoint doesn't exist
+        // mountpoint doesn't exists
```

Comment:
This is a comment regression, reverting a fix from #1317

Reply:
I noticed because it got in the way of the patch from this PR applying cleanly on top of 2.4.5.